tcf_website/static/base/navbar.css

```diff
@@ -0,0 +1,97 @@
+/*
+    Any css that involves the navbar should go in this file!
+    This way things are kept insolated and de-coupled from one
+    another, making components more maliable and maintainable.
+*/
+
+.navbar {
+    flex-wrap: nowrap;
+}
+
+.tCF {
```

Comment:
thoughts on renaming to `logo-container` ?

Reply:
👍

Reply:
I renamed it